frontend/src/components/Components/Search/presenter.jsx

```
@@ -0,0 +1,58 @@
+import React from 'react'
+import {InputGroup, Menu, MenuItem, Spinner, Switch, Tooltip} from '@blueprintjs/core'
+import './style.css'
+
+export default function search(props) {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)